### DISCUSSION TITLE
any breaking change on the upgrade path from 1.44.3 to 2.0.3 ?

### DISCUSSION THREAD

**Discussion by marcelmfs** [netdata/netdata#19108] [Q&A]
Hello Community!

This week we had a big uptick on our central Netdata, and while I could just try to bring it back down by adding more computing resources I'm thinking about going to latest, since it's been a while since I upgraded the central instance for the last time.

To my surprise it's now on v2, kudos to you all on the milestone! 🥳 

As a rule of thumb when I see major version bumps I always read on the Change Log for breaking changes from the previous major release but couldn't find any special remark.

And instead of going through hours of google searching issues/forums I decided to just ask, should I be doing anything differently this time? Are there any attention points I should stop over and be careful about? 

Or can I just go ahead and drop-in v2 on top of v1.44.3 and it would automagically work as usual? 

Thank you for your time in advance and keep up the good work!
<3 Netdata

**Comment by ilyam8** [comment]
Hi, @marcelmfs. There are breaking changes, yes. You need to check the deprecation notice for every tag (e.g., [v1.45.0 deprecation notice](https://github.com/netdata/netdata/releases/tag/v1.45.0#v1450-deprecation-notice)).
